apps/authentication/models.py

```diff
@@ -113,6 +113,7 @@ class OnlineUser(AbstractUser):
     linkedin = models.URLField(_(u"linkedin"), blank=True, null=True)
     gender = models.CharField(_(u"kjønn"), max_length=10, choices=GENDER_CHOICES, default="male")
     bio = models.TextField(_(u"bio"), blank=True)
+    saldo = models.PositiveSmallIntegerField(_(u"saldo"), default=0)
```

Comment:
It's not unheard of not having an integer amount of money in an account. Maybe reconsider field type?